docs/development/performance/index.rst

```diff
@@ -78,6 +80,8 @@ OZ version        Role      Median time (in ms)
 1.14.0 (fuzzy on) Regular   3400
 1.17.0 (fuzzy on) Superuser 3300
 1.17.0 (fuzzy on) Regular   4000
+1.18.0 (fuzzy on) Superuser 1900
+1.18.0 (fuzzy on) Regular   2300
```

Comment:
@joeribekker No idea why this is faster, we didn't do any performance fixes in this release. I think we should make this automated, because doing it manually makes things too flakey (https://github.com/maykinmedia/open-api-framework/issues/20)